JavaProHome/src/main/java/homework8/ProductManager.java

```diff
@@ -0,0 +1,99 @@
+package homework8;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class ProductManager {
+
+    private int lastProductNumber = 0;
+    private List<Product> products;
+
+    public ProductManager() {
+        products = new ArrayList<>();
+    }
+
+    public void addProduct(Product product) {
+        product.setId(++lastProductNumber);
+        products.add(product);
+    }
+
+    public List<Product> getProductsByTypeWithPriceHigherThan(
```

Comment:
```suggestion
    public List<Product> getProductsByTypeWithPriceGT(
```

https://stackoverflow.com/questions/21317399/enum-for-gt-gte-eq-lt-lte-operators